### ISSUE TITLE
Converter eats literal curly Mustache-esque curly braces

### ISSUE THREAD

**Issue by alanhogan** [twilson63/html2haml#20]
Steps to reproduce:
- Include the string `{{foo}}` in your HTML, as content, or in attribute values
- Convert

Expected:
- `{{foo}}` survives

Actual:
- `{{foo}}` disappears

**Comment by twilson63** [comment]
Yep, the convert is only meant to parse HTML or ERB to Haml, not sure if it was ever meant to support mustache like templates.

**Comment by alanhogan** [comment]
Hmm, right. Well, the thing is, it was for use _client-side_, so in at least my use case, I’d simply want them **preserved**, as opposed to converted or removed. 

Now, I personally dislike AngularJS and think it breaks standards and breaks with good practice, _but_ this is going to be common for AngularJS sites.

**Comment by twilson63** [comment]
Hey Alan,

I will see what I can do, there may be a way to wrap them in cdata tags or
something to preserve them, then unwrap them when the haml conversion is
completed.

Thanks

Tom

On Mon, Sep 16, 2013 at 8:07 PM, Alan Hogan notifications@github.comwrote:

> Hmm, right. Well, the thing is, it was for use _client-side_, so in at
> least my use case, I’d simply want them _preserved_, as opposed to
> converted or removed.
> 
> Now, I personally dislike AngularJS and think it breaks standards and
> breaks with good practice, _but_ this is going to be common for AngularJS
> sites.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/twilson63/html2haml/issues/20#issuecomment-24555085
> .

## 

Tom Wilson
Jack Russell Software Company Division of CareKinesis
494 Wando Park Blvd
Mount Pleasant, SC  29464
Phone: 843-881-2171
Email: tom@jackhq.com
Web: http://www.jackhq.com
Calendar:
http://www.google.com/calendar/embed?src=tom%40jackrussellsoftware.com&ctz=America/New_Yorkhttp://www.jackhq.com/calendar

This e-mail may contain information that is confidential, privileged or
otherwise protected from disclosure by the Health Insurance Portability and
Accountability Act (HIPAA) and other state and federal laws.  This
information is intended only for the individual names above.  Any review,
use disclosure or dissemination of this material is strictly prohibited.
If you receive this information in error, please notify CareKinesis
immediately at 888-974-2763 and delete the original at once.

**Comment by alanhogan** [comment]
:smile: You’re the man
